tests/test_move.py

```diff
@@ -60,3 +82,91 @@ def test_move_dir(self):
         if self.preserve_time:
             dst_file2_info = dst_fs.getinfo("bar/baz.txt", namespaces)
             self.assertEqual(dst_file2_info.modified, src_file2_info.modified)
+
+
+class TestMove(unittest.TestCase):
+    def test_move_file_tempfs(self):
+        with open_fs("temp://") as a, open_fs("temp://") as b:
+            dir_a = a.makedir("dir")
+            dir_b = b.makedir("subdir")
```

Comment:
Now that you've "fixed" it so that `a` and `b` are independent FSes, I think you probably don't need these subdirectories?

Reply:
not needed, but it's nice to test the base / common logic a bit?

Reply:
Fair enough. Might be nice to use more helpful variable-names than `a`, `b`, `dir_a` and `dir_b` though? :wink:

Reply:
Done